.github/workflows/docs.yml

```diff
@@ -0,0 +1,36 @@
+name: API Docs
+
+on:
+  push:
+    branches:
+      - main
+    pull_request:
+
+jobs:
+  docs:
+    name: Sphinx
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Build docs
+        run: |
+          pip install virtualenv
+          make apidocs
+
+      - uses: peaceiris/actions-gh-pages@v3.7.3
```

Comment:
Do we have a list of vetted actions we are allowed to use in our repos?

Reply:
No but I got this directly from the discuss-docs channel. It is being used in a bunch of our current repos.